networkx/drawing/nx_pylab.py

```diff
@@ -522,8 +522,11 @@ def draw_networkx_edges(
         Also, `(offset, onoffseq)` tuples can be used as style instead of a strings.
         (See `matplotlib.patches.FancyArrowPatch`: `linestyle`)
 
-    alpha : float or None (default=None)
-        The edge transparency
+    alpha : float or array of floats (default=None)
+        The edge transparency.  This can be a single alpha value,
+        in which case it will be applied to all the nodes of color. Otherwise,
```

Comment:
Since this is in `draw_networkx_edges`, shouldn't it specify the alpha value of the edges? In which case maybe wording should be something like:

```suggestion
        in which case it will be applied to all specified edges. Otherwise,
```

Reply:
Oops, missed this, thanks!